hivemind/optim/optimizer.py

```diff
@@ -197,6 +197,8 @@ def __init__(
         shutdown_timeout: float = 5,
         verbose: bool = False,
     ):
+        self._parent_pid = os.getpid()
+
```

Comment:
Before moving this line, there was an extra exception below in case of any meaningful exception in `__init__`:

```python
Exception ignored in: <function Optimizer.__del__ at 0x7f282b8db1f0>                                                                                                              
Traceback (most recent call last):                                                                                                                                                
  File "/home/hivemind/borzunov/hivemind/hivemind/optim/optimizer.py", line 764, in __del__                                                                                       
    if self._parent_pid == os.getpid() and self.is_alive():                                                                                                                       
AttributeError: 'Optimizer' object has no attribute '_parent_pid'
```